index.bs

```diff
@@ -305,18 +253,25 @@ spec: mimesniff; urlPrefix: https://mimesniff.spec.whatwg.org/#
 
       <p>
         The <dfn for='VideoConfiguration' dict-member>contentType</dfn> member
-        represents the MIME type of the video track.
+        represents the [=MIME type=] of the video track.
       </p>
 
       <p>
         To check if a {{VideoConfiguration}} <var>configuration</var> is a
         <dfn>valid video configuration</dfn>, the following steps MUST be run:
         <ol>
           <li>
-            If <var>configuration</var>'s {{VideoConfiguration/contentType}} is
-            not a <a>valid video MIME type</a>, return <code>false</code> and
+            Let |mimeType| be the result of running [=parse a MIME type=]
```

Comment:
Optional: Do the "easier" checks 4 and 5 first, before invoking _parse a MIME type_.   I am under the impression the order of the checks does not matter.